packages/blocks/src/api/validation.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import Tokenizer from 'simple-html-tokenizer/dist/es6/tokenizer';
```

Comment:
I'm guessing by your measure of the size difference, it has not regressed (possibly because of better ES module support than they had previously†), but the reason it was originally written this way was to avoid the side effect of importing the _very very large_ character reference set.

https://github.com/tildeio/simple-html-tokenizer/blob/9a894bd75545e107638272f3e1e812933815953f/src/index.ts#L2-L4

https://github.com/tildeio/simple-html-tokenizer/blob/master/src/generated/html5-named-char-refs.ts

_† I'd forgot I made this pull request: https://github.com/tildeio/simple-html-tokenizer/pull/49_ 😅

Reply:
It would explain why it works so well now 😍